packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1030,7 +1023,6 @@ export class GarbageCollector implements IGarbageCollector {
 			lastSummaryTime: this.getLastSummaryTimestampMs(),
 			headers: headerData,
 			requestUrl: request?.url,
-			requestHeaders: JSON.stringify(request?.headers),
```

Comment:
I feel like this was here for a reason. It shows what was actually provided by caller, v. whatever headerData was modified for implementing logic.

Reply:
We never looked at it and I doubt we will. The only case where headerData and headers can be different is when requesting for sub-data store objects. Do we need to differentiate that case?